kDrive/UI/View/Header view/FilesHeaderView.swift

```diff
@@ -73,6 +74,10 @@ class FilesHeaderView: UICollectionReusableView {
         )
         let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didTapOnCard))
         uploadCardView.addGestureRecognizer(tapGestureRecognizer)
+
+        selectView = SelectView.instantiate()
```

Comment:
So I guess your intent is to keep the `Class.instantiate()` pattern only for xib related views ?

Reply:
Yes for now I didn't to touch the XIBs 👍